gprofiler/profilers/java.py

```diff
@@ -280,6 +280,11 @@ def get_ap_version() -> str:
 
 
 T = TypeVar("T", bound="AsyncProfiledProcess")
+_MEM_INFO_LOG_RE = re.compile(
```

Comment:
Give permalink reference to async-profiler upstream repo where this format is defined

Reply:
done